src/poojaRDec19/Assignment3/Q4.java

```diff
@@ -0,0 +1,31 @@
+package poojaRDec19.Assignment3;
+
+//4) WAP to find sum of user given number. a. Input:- 159753 b. Sum of given number:- 30 
+import java.util.*;
+public class Q4 {
+	int m,sum,n;
+	void sum()
```

Comment:
HI @PoojaRDec19 
As we discuss always write parameterized method
And Do formatting your code using 
Ctrl+Shift+f